content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -311,8 +311,7 @@ up to 3 replicas, as well as scaling down the old ReplicaSet to 0 replicas.
 ### Rollover (aka multiple updates in-flight)
 
 Each time a new Deployment is observed by the Deployment controller, a ReplicaSet is created to bring up
-the desired Pods. If the Deployment is updated, the existing ReplicaSet that controls Pods whose labels
-match `.spec.selector` but whose template does not match `.spec.template` are scaled down. Eventually, the new
+the desired Pods. If the Deployment is updated, the existing ReplicaSet with its label(s) in `.spec.selector` matching the pod's template label(s) in `.spec.template` are scaled up but those pod's template label(s) in `.spec.template` not matching with the ReplicaSet's labels in `.spec.selector` are scaled down. Eventually, the new
```

Comment:
👋 @MIhirMishra .
This change does not seem clear, especially this section of the change:
`but those pod's template label(s) in `.spec.template` not matching with the ReplicaSet's labels in `.spec.selector` are scaled down`

Reply:
Hi @MIhirMishra .
Thanks for looking into this section. I agree that the content in this section could be improved.
I will open an issue to indicate that this page and section needs updating.
At any time, you can reopen this PR. Thank you.